### PR TITLE
Link: fallback to using href as text content

### LINKED ISSUE #721: Link component should use href as text if no children are provided
If an href is provided to the `Link` component, but no child text is present, then the href value should be displayed:
```
<Link href="https://google.com" />
```

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Fixes #721

Change-type: minor
Signed-off-by: Lucian <lucian.buzzo@gmail.com>

### CODE REVIEW

File: __tests__/components/Link.spec.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+    const url = 'https://google.com'
+    const component = mount(
+      <Provider>
+        <Link href={url} />
+      </Provider>
+    )
+
+    console.log(component.html())
+
+    expect(component.find(`a [href="${url}"]`)).toHaveLength(1)
```
Comment (sradevski) [inline __tests__/components/Link.spec.js]: NIT: I think this doesn't really test the spec, as it simply checks if there is an anchor with a `href` set to `url`. I think it should be something like ```expect(component.find(`a [href="${url}"]`)[0].text()).toEqual(url)```

Comment (LucianBuzzo) [reply __tests__/components/Link.spec.js]: @sradevski Yep you're totally right! This was a clear mistake, I'll fix it.

File: __tests__/components/Link.spec.js [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+
+  it('Should display the href prop if a child is not provided', () => {
+    const url = 'https://google.com'
+    const component = mount(
+      <Provider>
+        <Link href={url} />
+      </Provider>
+    )
+
+    console.log(component.html())
```
[the remark this answers is not in the document]
Comment (LucianBuzzo) [reply __tests__/components/Link.spec.js]: Yep, I'll fix this

**Comment by sradevski** [comment]
@LucianBuzzo I think this one is good to go, would you mind rebasing and merging it? Just so it doesn't get outdated and forgotten :smile:

**Comment by LucianBuzzo** [comment]
@sradevski I'm focusing on https://github.com/balena-io-modules/rendition/pull/688 and then I'll update and merge this one

**Comment by LucianBuzzo** [comment]
@resin-ci retest

**Pull request merged**